cmd/security-profiles-operator/main.go

```diff
@@ -506,7 +506,7 @@ func runDaemon(ctx *cli.Context, info *version.Info) error {
 func runBPFRecorder(_ *cli.Context, info *version.Info) error {
 	const component = "bpf-recorder"
 	printInfo(component, info)
-	return bpfrecorder.New("", ctrl.Log.WithName(component), true, false).Run()
+	return bpfrecorder.New("", ctrl.Log.WithName(component), true, true).Run()
```

Comment:
All in favor of this, but do we have any (rough) estimations on additional memory usage? The number of syscalls is naturally limited, the number of accessed file paths not so much. 🫣

Reply:
This is just enabling the recorder into the operator. If no `ProfileRecording` CR is created, there will not be any recording triggered. 

I still need to add some integration tests and do more testing.

Also in the past we did some memory optimisation for operator itself when it comes to big clusters with a lot of nodes. It used to run out of memory. If I understand correctly, I think you are referring to the memory usage of the eBPF recorder when a recording is performed. 

Do you have any numbers for apparmor profiles when they are recorded form the CLI?

Reply:
> If no `ProfileRecording` CR is created, there will not be any recording triggered.

Ok. If we only persist things for the containers for which a `ProfileRecording` CR is created, I am much less worried!

> Do you have any numbers for apparmor profiles when they are recorded form the CLI?

This is highly specific on the workload. For the workloads I tested so far it's fairly negligible (in the order of 1kb/container), but it's a very different story if we have a container that continuously writes temporary files with random names. That would just grow without bounds at the moment I think. There really isn't a great way to fix that AFAIK, so we probably just want to accept the risk. :)